launchlib/config.go

```diff
@@ -267,6 +267,10 @@ func parseCustomConfig(yamlString []byte) (PrimaryCustomLauncherConfig, error) {
 				"subProcess config %s", name)
 		}
 	}
+	if config.Experimental.ContainerV2 == nil {
+		var trueVal = true
+		config.Experimental.ContainerV2 = &trueVal
```

Comment:
I've seen this pattern elsewhere, but it would be nice if I could make this one line.

Reply:
I don't think there's anything in the standard library for this :( but there are some pretty small deps for doing something like [this](https://go.dev/play/p/vwkCLlBqQSm):
```golang
package main

import (
    "fmt"
    "k8s.io/utils/ptr"
)

func main() {
    a := ptr.To(false)
    fmt.Printf("%T %v", a, *a)
}

```

Reply:
I see. I'm leaning towards not adding the dep since we use this in one place in the implementation (and also once in the tests).